README.md

```diff
@@ -4,6 +4,8 @@ Small server wrapper around Z-Wave JS to access it via a WebSocket.
 
 ## Trying it out
 
+Keep in mind that on production you want to execute `npm run build` and use `node` instead of `ts-node` for all below commands.
```

Comment:
That's not correct. If you install this using the command `npm install -g @zwave-js/server` you will be able to run this by simply call `zwave-server /dev/ttyACM0` and `zwave-client` taken from: https://github.com/zwave-js/zwave-js-server/blob/master/package.json#L7